less/bootstrap-fixes.less

```diff
@@ -213,4 +213,30 @@ label.checkbox, label.radio, label.checkbox-inline, label.radio-inline {
 
 .popover button.close {
   opacity: 1;
-}
\ No newline at end of file
+}
+
+/* Contrast fixes ID-347 */
+
+blockquote small {
+  color: @text-muted;
```

Comment:
Better to override `@blockquote-small-color` in variables.less so it just updates the Boostrap default, and also applies to any other rules it's used in